src/Validator.php

```diff
@@ -167,8 +167,12 @@ public static function validateDoctrineFieldName(string $name, ManagerRegistry $
         return $name;
     }
 
-    public static function validateEmailAddress(string $email): string
+    public static function validateEmailAddress(?string $email): string
     {
+        if (null === $email) {
+            throw new RuntimeCommandException('Email should not be null.');
+        }
+
```

Comment:
I don't think this is needed as the `filter_var()` call just below is doing the same thing.